### ISSUE TITLE
Top and bottom positioning

### ISSUE THREAD

**Issue by TomberWolf** [ahkohd/switch-desktop#49]
Could you add top and bottom positioning please?

**Comment by ahkohd** [comment]
@TomberWolf  Noted! Next major release.

**Comment by ghost** [comment]
Да, очень не хватает нижнего скажем расположения. Панель задач использую сверху, а Ваш док хотелось бы снизу.
